openslides/agenda/static/templates/agenda/item-detail.html

```diff
@@ -31,26 +31,53 @@ <h3 os-perm="agenda.can_manage" translate>Comment</h3>
 </div>
 
 
-<h3 translate>List of speakers</h3>
+<h3 translate>List of speakers
```

Comment:
Please put only the headline into the \<h3> tag and the links to open and close the list after it.